hackerproject.js

```diff
@@ -6,3 +6,22 @@
  * @param {string} str 
```

Comment:
The function looks great, but the commit message doesn't follow our naming convention, can you please give appropriate commit message.